data/data-home.json

```diff
@@ -96,7 +119,7 @@
             },
             {
                 "name":"Parth Sharma",
-                "designation":"Student",
+                "designation":"Student ex-VP",

```

Comment:
Come on? @parthsharma2 seeing this?

Reply:
![](https://cultofthepartyparrot.com/parrots/hd/spyparrot.gif)